collectd/logstash.conf

```diff
@@ -8,7 +8,7 @@ LoadPlugin network
     IgnoreSelected false
 </Plugin>
 <Plugin network>
-    <Server "logstash" "5000">
+    <Server "logstash">
     </Server>
```

Comment:
Yup, that's default

> The optional second argument specifies a port number or a service name. If not given, the default, 25826, is used.

https://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_network